#src/index.html

```diff
@@ -4,10 +4,29 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Discover</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Mulish:wght@400;700&family=Playfair+Display:wght@700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-    
+    <header>
+        <div class="container container--wider">
+            <div class="row">
+                <img src="img/logo.png" alt="logo" class="col-md-1 logo">
+                <nav class="nav offset-md-4 col-md-7">
+                    <ul class="nav__list">
+                        <li class="nav__item nav__item--black"></li>
```

Comment:
Give classes based on functionality, not on theme.
In this case link is dark, because it indicates the current page.
```suggestion
                        <li class="nav__item nav__item--current"></li>
```
or more conventionally
```suggestion
                        <li class="nav__item nav__item--active"></li>
```